schemas/alternate-registration-schema.json

```diff
@@ -0,0 +1,93 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "A representation of multiple registrations for the same entity",
+  "type": "object",
+  "properties": {
+    "data_type": {
+      "enum": [
+        "alternate-registration"
+      ]
+    },
+    "alternate_entities": {
+      "description": "TBC",
+      "type": "array",
+      "items": {
+        "$ref": "includes/entity.json",
+        "minLength": 2
+      }
+    },
+    "alternate_registration_start_date": {
+      "description": "date when alternate registration started",
+      "type": "string",
+      "format": "date"
+    },
+    "alternate_registration_end_date": {
+      "description": "date when alternate registration ended",
+      "type": "string",
+      "format": "date"
+    },
+    "publication_date": {
+      "description": "TBC",
+      "type": "string",
+      "format": "date"
+    },
+    "start_date": {
+      "description": "date when alternate registration was valid from",
+      "type": "string",
+      "format": "date"
+    },
+    "start_date_type": {
+      "enum": [
+        "at",
+        "before",
+        "after"
+      ]
+    },
+    "sample_date": {
+      "description": "date when alternate registration was sampled",
+      "type": "string",
+      "format": "date"
+    },
+    "retrieved_at": {
+      "description": "date when alternate registration was retrieved",
+      "type": "string",
+      "format": "date"
+    },
+    "source_url": {
+      "description": "URL of the source of the data (e.g. download URL), or if there is not persistent URL the page from which it can be found (e.g. search page)",
+      "type": "string"
+    },
+    "confidence": {
+      "description": "Confidence in accuracy of data",
+      "enum": [
+        "HIGH",
+        "MEDIUM",
+        "LOW"
+      ]
+    }
+  },
+  "additionalProperties": false,
+  "required": [
+    "data_type",
+    "alternate_entities",
+    "retrieved_at",
+    "confidence"
+  ],
+  "anyOf": [
+    {
+      "required": [
+        "start_date"
+      ]
+    },
+    {
+      "required": [
+        "end_date"
```

Comment:
Just noticed we still have this, but no `end_date` property! Odd that json-schema doesn't complain.